vignettes/release-checklists.qmd

```diff
@@ -0,0 +1,110 @@
+---
+title: "Release Checklists"
+description: >
+  Reference for how to perform R package releases in the Hubverse
+vignette: >
+  %  \VignetteIndexEntry{Release Checklists}
+  %  \VignetteEngine{quarto::html}
+  %  \VignetteEncoding{UTF-8}
+knitr:
+  opts_chunk: 
+    collapse: true
+    comment: '#>'
+---
+
+## Release Checklist
+
+ - [ ] Create a new branch from `main` called `<author>/release/<X.Y.Z>` (e.g. `ls/release/0.1.6`)
+ - [ ] Update version number in DESCRIPTION and check that the `Remotes:` is
```

Comment:
maybe we could add a manual check to make sure that the author list has been updated based on contributions since the last version?

Reply:
That's a good suggestion! I'll make sure to add it!